### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by spokenbird**
[#182186008]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,45 +45,23 @@ can fill out screens with their basic info, upload supporting documents, then su
 Upon submission, they receive an email with a filled-in
 application PDF. The entire experience is in both English and Spanish.
 
-The `form-flow` Java library is included in `build.gradle` along with all other dependencies. The codebase for the `form-flow`
-package is [open source](https://github.com/codeforamerica/form-flow).
+The `form-flow` Java library is included in the applications `build.gradle` along with all other
```
Comment (vrajmohan) [inline README.md:48]: application's

File: README.md [hunk 2]
```diff
@@ -42,45 +45,23 @@ can fill out screens with their basic info, upload supporting documents, then su
...
-```
+To experience the starter-app for yourself, you can visit:
+[https://forms-starter.cfa-platforms.org/](https://forms-starter.cfa-platforms.org/)
 
 ## Static Pages
 
-Unlike Screens, Static Pages are HTML content and are not part of a flow. Detailed information
-about static pages can be found in
-the [form-flow library documentation.](https://github.com/codeforamerica/form-flow#static-pages)
+The starter-app implements some purely static pages which defer from screens in that they take in
```
Comment (vrajmohan) [inline README.md:59]: s/defer/differ

File: README.md [hunk 3]
```diff
@@ -132,8 +120,71 @@ client has indicated that they moved to the USA in the last year. If they have,
...
 ### Conditions
 
+Conditions are used either to determine navigation between screens or to determine whether
+elements should be displayed on a screen. If you would like to learn more about creating your own 
+conditions, please see the [form-flow library documentation](https://github.com/codeforamerica/form-flow#conditions).
+
+#### HasHousehold
+
+The `HasHousehold` condition is an example of a condition that is used to determine navigation
+between screens. In the starter-app this condition is used to determine whether a user should be shown
```
Comment (vrajmohan) [inline README.md:134]: s/starter-app/starter app/

File: README.md [hunk 4]
```diff
@@ -132,8 +120,71 @@ client has indicated that they moved to the USA in the last year. If they have,
...
+should follow the `housemates` screen looks like this:
+
+```yaml
+  housemates:
+    nextScreens:
+      - name: housemateInfo
+        condition: HasHousehold
+      - name: income
+```
+The above YAML inidicates that the `housemateInfo` screen should be shown if the `HasHousehold`
```
Comment (vrajmohan) [inline README.md:155]: typo - indicates

File: README.md [hunk 5]
```diff
@@ -132,8 +120,71 @@ client has indicated that they moved to the USA in the last year. If they have,
...
+The `IncomeSelectedSelf` condition is an example of a condition that is used to determine what element
+to display in a screen. In the starter-app this condition is used to determine whether a user
+has indicated that they are entering income information about themselves or someone else. 
+
+If they have indicated that they are entering information about themselves, then the incomeTypes
+screen will show a header that says `What sources do you receive income from?`. If they 
+are entering income information for someone else, the header will instead read 
+`What sources does **persons name** receive income from?`.
+
+The condition is run within the incomeTypes screens thymeleaf template file and uses the
```
Comment (vrajmohan) [inline README.md:169]: s/screens/screen's/

File: README.md [hunk 6]
```diff
@@ -184,17 +235,17 @@ Live Reload is very helpful when making many changes to HTML templates, CSS, or
...
-### Contributing Live Templates to your App
+### Contributing Live Templates to Your App
 
 If you have created live templates with fragments which are specific to your application based on a
 starter app template, you can commit them to your repository. You will follow a similar pattern to
 create templates to what is outlined
 [in the form-flow library here.](https://github.com/codeforamerica/form-flow#contribute-new-live-templates)
 
-An example template which was set up using this process, starting from an html snippet is available
+An example template which was set up using this process, starting from a html snippet is available
```
Comment (vrajmohan) [inline README.md:245]: IMO, it should be "an".

File: README.md [hunk 7]
```diff
@@ -212,16 +263,19 @@ app's [build.gradle](build.gradle) to pull in the local library, via this line:
...
 ## Actuator Endpoints
 
 The [Actuator](https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html) is a
-feature Spring Boot provides to monitor and interact with your production
-ready application. It opens endpoints that can be queried to get information
+feature Spring Boot provides to monitor and interact with your application. It opens endpoints that can be queried to get information
 about your application, like health and build information.
 
-**This feature can be a large security concern if certain endpoints are open.**
+The starter-app, by default, enables full access to all actuator endpoints in the dev profile for
```
Comment (vrajmohan) [inline README.md:274]: s/starter-app/starter app

File: README.md [hunk 8]
```diff
@@ -374,7 +427,7 @@ datasource:
 
 # Troubleshooting IntelliJ
 
-## Application won't run after IntelliJ update
+## Application Won't Run After IntelliJ Update
```
Comment (vrajmohan) [inline README.md:430]: IMO, it should be "won't".

**Review by vrajmohan — APPROVED** [review]
Minor comments

File: README.md (hunk 6 above)
Comment (spokenbird) [reply README.md:245]: I thought the same but IntelliJ's spell checker recommended "a". I'm changing it to "an" anyways, hehe.

**Pull request merged**